main.py

```diff
@@ -1,11 +1,72 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_races() -> None:
+    with open("players.json", "r") as file:
+        players_info = json.load(file)
```

Comment:
let's open the file in the main function to not do it every time because it may be a large file with 10 gigabytes of data